yesod-auth/Yesod/Auth.hs

```diff
@@ -110,8 +117,27 @@ class (Yesod master, PathPiece (AuthId master), RenderMessage master FormMessage
     -- destination exists.
     logoutDest :: master -> Route master
 
+    -- | Perform authentication based on the given credentials.
+    --
+    -- Default implementation is in terms of @getAuthId@
```

Comment:
Slightly preferable to have single quotes around `getAuthId` so that haddock generates a hyperlink to the function name.